script/cpplint.js

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env node
+
+const { GitProcess } = require('dugite')
+const childProcess = require('child_process')
+const fs = require('fs')
+const klaw = require('klaw')
+const minimist = require('minimist')
+const path = require('path')
+
+const SOURCE_ROOT = path.normalize(path.dirname(__dirname))
+const LINTER_PATH = path.join(SOURCE_ROOT, 'vendor', 'depot_tools', 'cpplint.py')
+const ROOTS = ['atom', 'brightray'].map(token => path.join(SOURCE_ROOT, token))
+
+function isBootstrapped () {
```

Comment:
nit, but this small and only used once, so you can put
`const isBootstrapped = () => fs.existsSync(LINTER_PATH)` in `main`